.github/workflows/test.yaml

```diff
@@ -88,9 +88,9 @@ jobs:
         conda env update -n pypsa-eur -f ${{ env.env_file }}
         echo "Run conda list" && conda list
 
-    - name: Run snakemake test workflows
+    - name: Run snakemake test workflows and unit tests
```

Comment:
Can you put the unit tests in a second step?
This will make it easier to distinguish in the logs.

Reply:
sure thing! Thanks @lkstrp